### PR TITLE
Add option to table_annovar to output VCF.

### PR DESCRIPTION
**Pull request by jgoecks**
table_annovar can now produce an annotated VCF file in addition to a tabular file. This pull request enhances the table_annovar wrapper to produce a VCF file.

### CODE REVIEW

File: tools/table_annovar/table_annovar.xml [hunk 1]
```diff
@@ -34,13 +39,23 @@
...
+        #end if
 
-        ## Process annotated table to remove "NA" strings from numerical columns and copy to output.
-        ; cat output.${input.dbkey}_multianno.txt | python \$SCRIPT_PATH/replace_NA.py > ${output}
+        ## Post-processing: process annotated table to remove "NA" strings from numerical columns if
+        ## tabular. Copy to output.
+        #if str($out_format) == "tabular":
+            ; cat output.${input.dbkey}_multianno.txt | python \${SCRIPT_PATH}/replace_NA.py > ${output}
+        #else:
+            ; cp output.hg19_multianno.vcf ${output}
```
Comment (bgruening) [inline tools/table_annovar/table_annovar.xml]: ${input.dbkey} instead of hg19?

File: tools/table_annovar/table_annovar.xml [hunk 2]
```diff
@@ -34,13 +39,23 @@
...
+        #set operation = ','.join( $operation )
+
+        ## Annotate variants.
+        table_annovar.pl ${tab_anno_input} ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring '.' -buildver ${input.dbkey} --outfile output
 
-        ## Reduce variants.
-        ; table_annovar.pl input.avinput ${__get_data_table_entry__('annovar_indexes', 'dbkey', $input.dbkey, 'path')} -protocol ${protocol} -operation ${operation} -nastring NA -buildver ${input.dbkey} --outfile output
+        ## Add option to consume/produce VCF.
+        #if str($out_format) == "vcf":
+            --vcfinput
```
Comment (bgruening) [inline tools/table_annovar/table_annovar.xml]: I'm not familiar with this tool, but is `--vcfinput` correct, not `--vcfoutput`?

Comment (jgoecks) [inline tools/table_annovar/table_annovar.xml]: Yes, this is correct: http://annovar.openbioinformatics.org/en/latest/misc/accessory/#table_annovar-automated-execution-of-multiple-annotation-tasks

File: tools/table_annovar/table_annovar.xml (hunk 1 above)
Comment (jgoecks) [inline tools/table_annovar/table_annovar.xml]: Fixed, thanks.

**Comment by bgruening** [comment]
Anyone else? Speak Now or Forever Hold Your Piece.

**Comment by jennaj** [comment]
The option -vcfinput is problematic. See issue https://github.com/galaxyproject/tools-devteam/issues/323

**Pull request merged**